tipg/db.py

```diff
@@ -70,15 +70,13 @@ async def __call__(self, conn: asyncpg.Connection):
 
 async def connect_to_db(
     app: FastAPI,
-    settings: Optional[PostgresSettings] = None,
+    settings: Optional[PostgresSettings] = PostgresSettings(),
```

Comment:
We should avoid!

Reply:
What is wrong with this?

Reply:
we should not use mutable in function parameters